CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.3.0 (2017-11-29)
 - Remove Elasticsearch 1.x and earlier code paths
 - Fix CI and configure an Elasticsearch 5.6 build
+- BREAKING: Rename `Index#optimize` to `Index#forcemerge`
```

Comment:
This change will not be included in the 2.3.0 release. The goal of the 2.3.0 release is to remove the ES 1.X code paths and make the library compatible **only** with ES 2.X.

This change will go out in the next release.